eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+import nx from '@nx/eslint-plugin';
```

Comment:
These linting rules are not currently applied - we could apply them for now and modify as we go.